sites/mosul/configs/openmrs/initializer_config/liquibase/liquibase.xml

```diff
@@ -75,6 +88,13 @@
 
     <!-- Retire CIEL concept for Nystatin -->
     <changeSet id="delete_ciel_concept_nystatin" author="vasharma05">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT CASE WHEN COUNT(*) > 0 THEN 1 ELSE 0 END
+                FROM concept
+                WHERE uuid = '80945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
```

Comment:
These changesets should only run when the CIEL's Nystatin drug concept exists, else it shouldn't.